lambdas/embedding/handler.py

```
@@ -0,0 +1,258 @@
+    logger.info("Processing %d messages", len(records))
+
+    datastore = get_datastore()
+    embedder = get_embedding_generator()
+    failures = []
+
+    try:
+        for record in records:
+            message_id = record["messageId"]
+            try:
+                process_message(record, datastore, embedder)
+            except (ProcessingError, Exception) as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)